lib/rubocop/cop/style/lambda.rb

```diff
@@ -38,6 +38,19 @@ def lambda_length(block_node)
 
           end_line - start_line
         end
+
+        def autocorrect(node)
+          expr = node.ancestors.first.loc.expression
+
+          @corrections << lambda do |corrector|
+            src = if node.loc.expression.source == 'lambda'
+                    expr.source.sub(/^lambda\s+\{\s*\|([^\|]+)\|/, '->(\\1) {')
```

Comment:
Good point. Suddenly I'm not so sure about the regex strategy anymore. :) Do I understand correctly that you're recommending me to add the new message within the scope of this issue? Sure, if that's the case.